### DISCUSSION TITLE
Digikey oauth failing, not listening on port 8139

### DISCUSSION THREAD

**Discussion by aaronw2** [inventree/InvenTree#2883] [General]
I am not sure what I'm doing wrong, but whenever I attempt to authenticate with Digikey with my keys I get the following error in my web browser: {"ErrorCode" : "invalid_request", "Error" :"Invalid redirection uri https://localhost:8139/digikey_callback"}

As far as I can tell, nothing on my host system is listening on port 8139. Is there some configuration option or do I need to install and run something to enable the Digikey callback support?

**Comment by matmair** [comment]
Hi there @aaronw2
As far as I am aware we neither ship with an oauth callback nor on that port. From where are you getting this url and what are you trying to do?
I am sure we will find a way to guide you into the right way, we will need some information about the goal and what architecture you had in mind. For callbacks to work you generally need the server to be reachable online - which might pose a security problem.

**Reply by aaronw2** [reply]
Never mind, this is in ki-nTree which is failing to connect. As far as I can tell, Inventree is working fine.

**Reply by matmair** [reply]
Ah, that makes sense. They have a separate repo with info's about config etc from this one, I am suture @eeintech will be able to assist you with further information.
